recipes/pybloom_live/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "pybloom_live" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 837550da7c82adb14b901c56c3e131ad269250cd19850a53671e878bd0626c68
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Could it be ``noarch`` ?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```